cypress-tests/cypress/e2e/PaymentUtils/Adyen.js

```diff
@@ -69,6 +69,19 @@ export const connectorDetails = {
         },
       },
     },
+    PaymentIntentOffSession: {
+      Request: {
+        currency: "USD",
+        customer_acceptance: null,
+        setup_future_usage: "off_session",
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_payment_method",
+        },
+      },
+    },
```

Comment:
there's no room to re-use this? what i mean here is that can we re-use `PaymentIntent` instead of re-writing the similar object?

Reply:
We need to pass off_session and we can re-use this any test case which required setup_future_usage as off_session